addons/crm/tests/common.py

```diff
@@ -417,13 +417,21 @@ def _aggregate(fname):
                 values = [_find_value(lead, fname) for lead in leads]
             return values
 
+        def _get_address(fname):
+            for lead in leads:
+                if any(lead[field] for field in PARTNER_ADDRESS_FIELDS_TO_SYNC):
+                    return lead[fname]
+
         try:
             # merge process will modify opportunity
             yield
         finally:
             # support specific values caller may want to check in addition to generic tests
             for fname, expected in expected.items():
-                self.assertEqual(opportunity[fname], expected)
+                if expected is False:
```

Comment:
OOC, why this change ?

Reply:
So I can check that a field is falsy (in that case it might be `False` or `self.env['res.country']`
```
self.assertLeadMerged(self.lead_w_contact, leads, country_id=False):
```

Reply:
yes :)